src/helpers/checkInstantiateReversion.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 @paritytech/contracts-ui authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { ContractInstantiateResult } from 'types';
+
+export function checkInstantiateReversion(dryRunResult: ContractInstantiateResult | undefined) {
+  return dryRunResult?.result.isOk
+    ? dryRunResult.result.asOk.result.flags.toHuman().includes('Revert')
```

Comment:
`dryRunResult.result.asOk.result.flags` has no field/flag `hasReverted` or so? 

Not familiar pjs to deeply still and can't find anything in the docs. Not sure if this is somehow available. Just did a quick search.

https://github.com/polkadot-js/api/blob/ab3b162d7eb96cccef9e3ff057d4dbe96ffe01a3/packages/types/src/interfaces/contracts/types.ts#L182